lib/ui/pointer.dart

```diff
@@ -38,21 +38,49 @@ enum PointerChange {
   up,
 
   /// A pan/zoom has started on this pointer.
+  ///
+  /// This type of events will always have kind [PointerDeviceKind.trackpad].
   panZoomStart,
 
   /// The pan/zoom on this pointer has updated.
+  ///
+  /// This type of events will always have kind [PointerDeviceKind.trackpad].
   panZoomUpdate,
 
   /// The pan/zoom on this pointer has ended.
+  ///
+  /// This type of events will always have kind [PointerDeviceKind.trackpad].
   panZoomEnd,
 }
 
 /// The kind of pointer device.
 enum PointerDeviceKind {
   /// A touch-based pointer device.
+  ///
+  /// The most common case is a touch screen.
+  ///
+  /// When the user is operating with a trackpad on iOS (for example, Magic
+  /// Keyboard on iPad), clicking will also dispatch events with kind [touch] if
```

Comment:
Apple actually names [another completely different thing](https://www.apple.com/shop/product/MXQT2LL/A/magic-keyboard-for-ipad-pro-11-inch-3rd-generation-and-ipad-air-5th-generation-us-english-black?afid=p238%7CsbeH3YzGF-dc_mtid_1870765e38482_pcrid_569734192244_pgrid_104394128081_pntwk_g_pchan__pexid__&cid=aos-us-kwgo-btb--slid---product-) as Magic Keyboard...

Also I've just discovered that the Magic Trackpad might work on iPad too. So I think I'm just going to remove this parenthesis to avoid adding confusion.